src/moneyTracker.js

```diff
@@ -0,0 +1,63 @@
+function Moneytracker(date, credit, debit, balance) {
+  this.date = "";
+  this.credit = 0;
+  this.debit = 0;
+  this.balance = 0;
+  this.MIN_BALANCE = 0;
+  // this.displayMessage = "Welcome to Moneytracker";
+};
+
+var transactions = [];
+
+Moneytracker.prototype.deposit = function (money, dateOfTransaction) {
+  this.credit += money;
```

Comment:
Does Moneytracker need to store `credit` and `debit`? Or are those already stored elsewhere? When designing a program, it may be helpful to have as a high level thought, "How can I store each piece of information in only one place?"